### PR TITLE
servantes: fixes for arm64

### PR DESCRIPTION
**Pull request by landism**

### CODE REVIEW

**Review by nicksieger — APPROVED** [review]
Fixes it for me 👍

File: fortune/Dockerfile.dc [hunk 1]
```diff
@@ -1,19 +0,0 @@
...
-
-RUN go get github.com/golang/protobuf/protoc-gen-go
-
-ADD . /go/src/github.com/tilt-dev/servantes/fortune
-RUN cd /go/src/github.com/tilt-dev/servantes/fortune && make proto
-RUN go install github.com/tilt-dev/servantes/fortune
-
-ENV TEMPLATE_DIR /go/src/github.com/tilt-dev/servantes/fortune/web/templates
-
-ENTRYPOINT /go/bin/fortune
```
Comment (nicksieger) [inline fortune/Dockerfile.dc:19]: Not sure if these were separate dockerfiles for a reason, to test the `dockerfile=` arg maybe? I converted them to symlinks, we could also just delete the dockerfile arg and the *.dc files/links.

**Comment by landism** [comment]
> Not sure if these were separate dockerfiles for a reason, to test the dockerfile= arg maybe? I converted them to symlinks, we could also just delete the dockerfile arg and the *.dc files/links.

It looks like the difference is that it sets TEMPLATE_DIR, because for the k8s versions it is set in the deployment yaml.
However:
* other, similar services work fine without having a separate Dockerfile.dc (e.g., doggos)
* main.go just defaults to the same value anyway, so it's probably best to just delete Dockerfile.dc

**Comment by nicksieger** [comment]
Yeah, I added a missing TEMPLATE_DIR in the docker-compose file, so that is covered. Will push one more fix with them deleted.

**Pull request merged**